src/provider.ts

```diff
@@ -1,110 +0,0 @@
-/**
- * Internal dependencies
- */
-import { AwarenessManager } from '@/awareness-manager';
-import {
-	createPersistedCrdtDocMetaRecord,
-	getPersistedCrdtDocFromEntityMeta,
-	type EntityMetaRecord,
-} from '@/utilities/crdt';
-import { getHashForEntityRecord, getMetaFromEntityRecord } from '@/utilities/entity';
-import { Logger } from '@/utilities/logger';
-import { createWebSocketConnection, type WebSocketConnectionConfig } from '@/websocket-client';
-
-import type { CRDTDoc, ObjectData, SyncConfig } from '@wordpress/sync';
-import type { WebsocketProvider } from 'y-websocket';
-
-export class SyncProviderWithAwareness extends window.wp.sync.SyncProvider {
-	private logger: Logger = new Logger( 'provider' );
-
-	public constructor( config: WebSocketConnectionConfig ) {
-		super( [
-			createWebSocketConnection( {
-				...config,
-				onStatusChange: ( ...args ) => this.onProviderStatusChange( ...args ),
-			} ),
-		] );
-	}
-
-	public async createEntityMeta(
```

Comment:
Noting that this was moved to https://github.com/WordPress/gutenberg/commit/71cbfeed23cc2f8c7a8121c9ec3f9e040b3f9e1d, for anyone curious like me.